index.html

```diff
@@ -4651,6 +4651,14 @@ <h2>
           The <a>user agent</a> MUST NOT share information about the user with
           a developer (e.g., the shipping address) without user consent.
         </p>
+        <p>
+          One way that the API supports limited information sharing is through
+          the <a>redactList</a> associated with the
+          <a>onshippingaddresschange</a> event. This feature enables user
```

Comment:
Should probably change: 

> `associated with the <a>onshippingaddresschange</a> event.`
 
used in the creation of `<a>`physical addresses`</a>` throughout the API.

Then that should cover us for when I add the billing address stuff